packages/evm/src/opcodes/gas.ts

```diff
@@ -209,6 +250,16 @@ export const dynamicGasHandlers: Map<number, AsyncDynamicGasHandler | SyncDynami
         if (common.isActivatedEIP(2929) === true) {
           gas += accessStorageEIP2929(runState, keyBuf, false, common)
         }
+
+        if (common.isActivatedEIP(6800) === true) {
```

Comment:
Short nit: for Boolean evaluations the explicit ˋ=== trueˋ is not necessary and you are in fact also using both versions in the code atm. Can you remove for all occurrences?

Reply:
We should check if we can add this to the linter rules :+1: :smile: